docs/toc/CONTRIBUTE.rst

```diff
@@ -257,14 +257,14 @@ Information about environment setup for developers can be found in the
 Pull Request Pro Tips
 ~~~~~~~~~~~~~~~~~~~~~
 
--  **`Fork <http://guides.github.com/activities/forking/>`__ the repository** and
-   clone it locally. Connect your local repository to the original ‘upstream’ one
-   by adding it as a remote. **Pull in** changes from ‘upstream’ often to stay
-   up to date so that when you submit your pull request, merge
+-  `Fork <http://guides.github.com/activities/forking/>`__ **the repository**
+   and clone it locally. Connect your local repository to the original
+   ‘upstream’ one by adding it as a remote. **Pull in** changes from ‘upstream’
+   often to stay up to date so that when you submit your pull request, merge
    conflicts will be less likely. See more detailed instructions
    `here <https://help.github.com/articles/syncing-a-fork>`__.
--  **Create a `branch <http://guides.github.com/introduction/flow/>`__** for your
-   edits.
+-  **Create a** `branch <http://guides.github.com/introduction/flow/>`__ for
+   your edits.
```

Comment:
Please, up to here (also the Fork item), emphasize the actions instead of only the objects. For example, make "Create a branch" bold, or "Create" if the branch link can't be bold.